content/applications/marketing/email_marketing/unsubscriptions.rst

```diff
@@ -1,66 +1,117 @@
+click the :guilabel:`Send` button. Odoo then logs their reasoning for unsubscribing in the *Email
+Marketing* app for future analysis.
 
 Blacklist
-=========
+---------
 
-In addition to having the option to *Unsubscribe* from specific mailing lists, the recipient can
-also *Blacklist* themselves, meaning they will not receive *any* more emails.
+For a recipient to remove (i.e. blacklist) themselves from a mailing list during the unsubscription
+process, on the :guilabel:`Mailing Subscriptions` page, they must tick the checkbox beside the name
+of the mailing lists they wish to remove themselves from, and then click :guilabel:`Exclude Me`.
 
-.. note::
-   The mailing list has to be configured as *Public* in order to be visible for users.
+Upon clicking :guilabel:`Exclude Me`, Odoo informs the recipient they have been successfully
+blacklisted, with a message reading: :guilabel:`✔️ Email added to our blocklist`.
+
+.. image:: unsubscriptions/mailing-subscriptions-blocklist-question.png
+   :align: center
+   :alt: The blocklist question on the Mailing Subscriptions page that recipients see.
 
-To view a complete collection of blacklisted email addresses, navigate to :menuselection:`Email
+Beneath that, Odoo asks the former subscriber to :guilabel:`Please let us know why you want to be
+added to our blocklist`, and the user can proceed to choose the appropriate reason from a series of
+options presented to them.
+
+Once they've chosen the appropriate reason from the options presented to them, they can click the
+:guilabel:`Send` button. Odoo then logs their reasoning for blacklisting themselves in the *Email
+Marketing* app for future analysis.
+
+Blacklisted email addresses
+===========================
+
+To view a complete list of all blacklisted email addresses, navigate to :menuselection:`Email
 Marketing app --> Configuration --> Blacklisted Email Addresses`.
 
 .. image:: unsubscriptions/blacklisted-email-addresses.png
    :align: center
    :alt: View of the blacklisted email addresses page in Odoo Email Marketing.
 
-When a blacklisted record is selected from this page, Odoo reveals a separate page with that
-blacklisted recipient's contact information.
+When a blacklisted record is selected from this list, Odoo reveals a separate page with the
+recipient's contact information, along with the provided :guilabel:`Reason` why they chose to
+blacklist themselves.
 
 .. image:: unsubscriptions/blacklisted-contact-form.png
    :align: center
    :alt: View of a blacklisted contact detail form in Odoo Email Marketing.
 
-In the :guilabel:`Chatter` of this page, there's a time-stamped message, informing the user when
-that recipient blacklisted themselves (via a :guilabel:`Mail Blacklist created` log note).
+In the *chatter* of the blacklisted record page, there's a time-stamped message, informing the user
+when the recipient blacklisted themselves (via a :guilabel:`Mail Blacklist created` log note).
 

```

Comment:
Add a note for more context on what blacklisted emails mean for the database as a whole
```suggestion
.. note::
   Blacklisted emails are excluded from all marketing mailings, however, these emails can still
   receive transactional emails, such as order confirmations, shipping notifications, etc.

```